justfile

```diff
@@ -142,6 +146,7 @@ up *flags:
       ((c++)) && ((c==3)) && break
       sleep 5
     done
+    echo && sleep 1 && just ps
```

Comment:
Nit: I just remembered there's a [special syntax](https://github.com/casey/just#running-recipes-at-the-end-of-a-recipe) for running one recipe after another. Maybe we can use that?

```justfile
up *flags: && ps
```

Reply:
Oh right! We could, I think I'll need to leave the `sleep 1` here though.